tsdoc/package.json

```diff
@@ -19,6 +19,14 @@
   "homepage": "https://tsdoc.org/",
   "main": "lib-commonjs/index.js",
   "module": "lib/index.js",
+  "files": [
+    "lib-commonjs/*.js",
+    "lib-commonjs/*.js.map",
```

Comment:
These (and the patterns for `lib`) only match files in the root of the `lib-commonjs` folder. This project has files that are in nested folders.

Reply:
gotcha! will fix